scripts/vultr_reformat_single.sh

```
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Simple script to reformat a single Vultr server
+# Usage: ./vultr_reformat_single.sh
+
+# Set script directory and project root
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+PROJECT_ROOT="$(cd "$SCRIPT_DIR/.." && pwd)"
+
+# Load variables from .env.production
+ENV_FILE="$PROJECT_ROOT/.env.production"
+[ -f "$ENV_FILE" ] && source "$ENV_FILE"
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)